run.sh

```diff
@@ -122,6 +123,7 @@ create_cjk_subset() {
     echo "Generating font $output_font. Current time: $(date)."
     "$VIRTUAL_ENV"/bin/pyftsubset cached_fonts/"$input_font" \
                   --unicodes-file=cached_fonts/"$subset_codepoints" \
+                  --layout-features='*' \
```

Comment:
@dscorbett  `--glyph-names` is missing from here. Was it intentional?

Reply:
It wouldn’t hurt to include it, but the source CJK font doesn’t have glyph names.

Reply:
Alright